src/test/java/com/hillel/items_exchange/security/SecurityConfigIntegrationTest.java

```diff
@@ -41,14 +48,17 @@
     private static final String NOT_VALID_USERNAME = "nimda";
     private static final String NOT_VALID_PASSWORD = "drowssap";
 
+    @Value("${app.jwt.expiration.time.ms}")
+    private Long jwtTimeExpired;
```

Comment:
Why do you check the message that won't see any kind of user?

Reply:
What do you mean? This is not a message, this time until the token is valid. I think, we have to test this.

Reply:
Yea, it was misunderstanding...